beetsplug/discogs.py

```diff
@@ -47,6 +48,9 @@
 
     from beets.library import Item
 
+# Global logger.
+log = logging.getLogger("beets")
```

Comment:
It doesn't seem like it's being used?

Reply:
Ah, thank you, this was meant for #5897 but slipped in here.